pandas/io/formats/style.py

```diff
@@ -70,6 +70,15 @@ class Styler(object):
         a unique identifier to avoid CSS collisons; generated automatically
     caption: str, default None
         caption to attach to the table
+    index: bool, default True
+        determines if the index is rendered in the html output
```

Comment:
I realize these new parametrize should directly reflect the methods, so use ``hide_index``, ``hide_columns``